modules/custom/openy_activity_finder/modules/openy_prgf_activity_finder/config/install/core.entity_view_display.paragraph.activity_finder.default.yml

```diff
@@ -1,46 +1,69 @@
-uuid: 66abd84e-b230-4b82-a71c-6dd938affdf6
+uuid: ea357f82-795a-4807-8d4f-24d990adec41
 langcode: en
 status: true
 dependencies:
   config:
+    - field.field.paragraph.activity_finder.field_prgf_af_categ
+    - field.field.paragraph.activity_finder.field_prgf_af_categ_excl
     - field.field.paragraph.activity_finder.field_prgf_af_results_ref
     - field.field.paragraph.activity_finder.field_prgf_block
     - field.field.paragraph.activity_finder.field_prgf_hide_loc_select_step
     - field.field.paragraph.activity_finder.field_prgf_hide_program_categ
     - paragraphs.paragraphs_type.activity_finder
-  module:
-    - plugin
 id: paragraph.activity_finder.default
 targetEntityType: paragraph
 bundle: activity_finder
 mode: default
 content:
-  field_prgf_af_results_ref:
-    weight: 1
-    label: hidden
-    settings: {  }
+  field_prgf_af_categ:
+    weight: 4
+    settings:
+      match_operator: CONTAINS
+      size: 60
+      placeholder: ''
     third_party_settings: {  }
-    type: openy_json
+    type: entity_reference_autocomplete
     region: content
-  field_prgf_block:
-    weight: 0
-    label: hidden
-    settings: {  }
+  field_prgf_af_categ_excl:
+    weight: 5
+    settings:
+      match_operator: CONTAINS
+      size: 60
+      placeholder: ''
     third_party_settings: {  }
-    type: plugin_block_built
+    type: entity_reference_autocomplete
     region: content
-  field_prgf_hide_loc_select_step:
-    type: openy_json
-    weight: 2
+  field_prgf_af_results_ref:
+    weight: 3
+    settings:
+      match_operator: CONTAINS
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+    type: entity_reference_autocomplete
```

Comment:
could you use `openy_er` reference type? We should limit dependencies for decoupled architecture

Reply:
fixed